mod-configuration-server/src/test/java/org/folio/support/OkapiHttpClient.java

```diff
@@ -0,0 +1,125 @@
+package org.folio.support;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpClientRequest;
+
+import java.util.concurrent.CompletableFuture;
+
+public class OkapiHttpClient {
+  private final Vertx vertx;
+  private final String tenantId;
+  private final String userId;
+  private final String token;
+
+  public OkapiHttpClient(Vertx vertx, String tenantId, String userId, String token) {
+    this.vertx = vertx;
+    this.tenantId = tenantId;
+    this.userId = userId;
+    this.token = token;
+  }
+
+  public CompletableFuture<Response> post(String url, String jsonContent) {
+    HttpClient client = vertx.createHttpClient();
+
+    HttpClientRequest request = client.postAbs(url);
+    Buffer requestBuffer = Buffer.buffer(jsonContent);
+
+    final CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+
+    request.exceptionHandler(postCompleted::completeExceptionally);
+
+    request.setTimeout(5000);
+
+    request.handler(response ->
+      response.bodyHandler(responseBuffer -> {
+        final Response convertedResponse = new Response(
+          response.statusCode(),
+          responseBuffer.getString(0, responseBuffer.length()));
+
+        System.out.println(String.format("Received response: '%s':'%s'",
```

Comment:
Using logger.debug() allows to enable and disable verbose output (3 x System.out.println in this file).

Reply:
Indeed, replacing all of the printing in these tests with logging fell by the way side as the work got more involved. There is quite a lot of possible improvements to these tests.

I've introduced a logger and replaced all of the println statements with log.debug